.github/workflows/remote-copy.yml

```
@@ -184,31 +212,42 @@ jobs:
         id: manifest
         env:
           REMOTE_MANIFEST_FILE_LIST_PATH: ${{ vars.REMOTE_TMP_DIR }}/remote-copy-manifests-${{ github.run_id }}.log
+          LOCAL_MANIFEST_FILE_LIST_PATH: ./manifests-copied.log
+          MANIFEST_FILE_NAME: manifest.yaml
         # Generate manifests for files copied over in the earlier rsync job
         run: |
           ssh ${{ secrets.SSH_USER }}@${{ secrets.SSH_HOST_DATA }} -i ${{ steps.ssh.outputs.private-key-path }} /bin/bash <<'EOT'
           module use ${{ vars.YAMF_MODULE_PATH }}
           module load ${{ vars.YAMF_MODULE_NAME }}
 
           manifest_paths=()
-          while read -r path:
-            cd $(dirname $path)
-            manifest_dir=$(basename $path)
-            yamf add -n manifest.yaml --force $manifest_dir
-            manifest_paths+=("$manifest_dir/manifest.yaml")
-          done <<< "${{ steps.copy.outputs.paths }}"
+          for path in ${{ steps.copy.outputs.paths }}; do
+            echo "Path is $path"
+            manifest_dir=$(dirname $path)
+            cd $manifest_dir || exit
+            manifest_file=$(basename $path)
+            echo "Generating a manifest in $manifest_dir for $manifest_file"
```

```suggestion
            echo "Generating a manifest entry in $manifest_dir for $manifest_file"
```
Just being explicit that it isn't necessarily generating a manifest file, just an entry into a manifest file.